library/src/test/java/com/getstream/sdk/chat/viewmodel/messages/MessageListItemLiveDataTests.kt

```diff
@@ -0,0 +1,122 @@
+package com.getstream.sdk.chat.viewmodel.messages
+
+import androidx.arch.core.executor.testing.InstantExecutorExtension
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Observer
+import com.getstream.sdk.chat.adapter.MessageListItem
+import com.getstream.sdk.chat.createMessage
+import com.getstream.sdk.chat.view.messages.MessageListItemWrapper
+import io.getstream.chat.android.client.models.ChannelUserRead
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.User
+import org.amshove.kluent.shouldBeEqualTo
+import org.amshove.kluent.shouldNotBeEqualTo
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(InstantExecutorExtension::class)
+class MessageListItemLiveDataTests {
+
+    private val currentUser = User()
+    private lateinit var messages: MutableLiveData<List<Message>>
+    private lateinit var threadMessages: MutableLiveData<List<Message>>
+    private lateinit var typing: MutableLiveData<List<User>>
+    private lateinit var reads: MutableLiveData<List<ChannelUserRead>>
+
+    private lateinit var observer: TestObserver<MessageListItemWrapper>
+
+    private lateinit var sut: MessageListItemLiveData
+
+    @BeforeEach
+    fun setUp() {
+        observer = TestObserver()
+        messages = MutableLiveData(listOf())
+        threadMessages = MutableLiveData(listOf())
+        typing = MutableLiveData(listOf())
+        reads = MutableLiveData(listOf())
```

Comment:
Nit: could use [`emptyList`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/empty-list.html) for these

Reply:
Done